server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/api.ts

```diff
@@ -49,6 +49,7 @@ export function create(
 	tenantManager: core.ITenantManager,
 	storage: core.IDocumentStorage,
 	tenantThrottlers: Map<string, core.IThrottler>,
+	fluidAccessTokenGenerator: core.IFluidAccessTokenGenerator,
```

Comment:
Wonder if we want to make it optional in OSS?

Reply:
Yeah that's a good idea. I can create an endpoint only if this implementation is injected. However, I would still like to keep the tests as they test API calls from `Alfred` -> `A token generation API`.